apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/AssertUtils.java

```diff
@@ -0,0 +1,52 @@
+package gov.cms.bfd.server.war.utils;
+
+import static org.junit.Assert.fail;
+
+import java.util.Objects;
+
+public class AssertUtils {
```

Comment:
I love this class.  It might be better moved into `bfd-shared-utils` though so that the pipeline tests could use it too.

Reply:
We could do that.  It's similar to how junit5 does exception testing, for when you want something more than just "an exception of type X was indeed thrown"

Reply:
Possible solution of adding a `bfd-shared-test-utils` module to include this and other common testing classes was posed.